plugin/buycourses/src/service_catalog.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * List of services
+ * @package chamilo.plugin.buycourses
+ */
+/**
+ * Initialization
+ */
+use Doctrine\Common\Collections\Criteria;
+
+$cidReset = true;
+
+require_once '../../../main/inc/global.inc.php';
+var_dump($_SESSION['RECURRING']);
+$plugin = BuyCoursesPlugin::create();
+$includeSessions = $plugin->get('include_services') === 'true';
+$includeServices = $plugin->get('include_services') === 'true';

```

Comment:
It's weird to have the same value twice... Are you sure this is correct? Shouldn't one of them be "include_sessions"?